source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+namespace PlayFab
+{
+    typedef std::chrono::time_point<std::chrono::system_clock> TimePoint;
+
+    inline TimePoint GetTimePointNow()
+    {
+        return std::chrono::system_clock::now();
+    }
+
+    inline time_t GetPlayFabTimeTNow()
+    {
+        time_t now = std::chrono::system_clock::to_time_t(GetTimePointNow());
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_SWITCH)
+        return static_cast<Json::Int64>(now);
```

Comment:
The function returns a time_t, so why is it being explicitly cast to Json::Int64 just to be implicitly cast back?

Reply:
Scott had the same comment on the earlier failed PR.
I agree it's weird, and I don't know why.
I don't intend to change it in this PR, because this PR just moves this code from one file to another.
I'll file a bug about how it's weird for another PR.

Reply:
Bug Filed.